features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle_rac.sql

```diff
@@ -2328,6 +2328,7 @@ CREATE TABLE IDN_WEBHOOK_CHANNELS (
     ID INTEGER,
     WEBHOOK_ID INTEGER NOT NULL,
     CHANNEL_URI VARCHAR(255) NOT NULL,
+    CHANNEL_SUBSCRIPTION_STATUS VARCHAR(30) NOT NULL,
```

Comment:
We agreed to having nullable subscription status right. When it comes to http adapter we don't have a such a status to maintain

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6885/commits/fb575c056228c93d2221b4ea29c500a76d09311c